lib/matplotlib/widgets.py

```diff
@@ -1007,7 +1007,12 @@ def __init__(self, ax, labels, active=0, activecolor='blue'):
             The index of the initially selected button.
         activecolor : color
             The color of the selected button.
+        orientation : str
+            The orientation of the buttons: 'vertical' (default), or 'horizontal'.
         """
+        if orientation not in ['vertical', 'horizontal']:
+            raise ValueError("Invalid RadioButton orientation: %s" % orientation)
```

Comment:
Can you add what are the possible values here?
Can you also add a mock test to make sure it runs, as well as a test to make sure this exception is raised properly.

I also think we need to update the examples to show case this (examples/widgets/radio_buttons.py)

Reply:
We have a shortcut function to check options like this and raise an appropriate error now, [`cbook._check_in_list`](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/cbook/__init__.py#L2102)